README.md

```diff
@@ -96,6 +96,7 @@ You can access these pages on your computer using one of the following clients:
     [App Store](https://itunes.apple.com/us/app/tldt-pages/id1071725095?ls=1&mt=8)
 - [Haskell client](https://github.com/psibi/tldr-hs):
   `stack install tldr`
+  or `apt-get install tldr` on Debian and Ubuntu
```

Comment:
Why mention `Ubuntu` specifically here? It implies that other Debian-based distros (e.g. Linux Mint or [Deepin](https://www.deepin.org/en/)) are excluded. How about this?
```suggestion
  or `apt-get install tldr` on Debian-based distributions such as Ubuntu
```

Reply:
```suggestion
  or `apt-get install tldr` on Debian-based distributions
```
is enough

Reply:
That's true. I will commit your suggestions @sbrl, @zdroid.